Dockerfile

```diff
@@ -1,55 +1,62 @@
 # Stage 1 - Compile needed python dependencies
-FROM python:3.7-buster AS build
+FROM python:3.10-slim-bullseye AS backend-build
```

Comment:
I receive 400 now when for all requests to the landing page with the running docker container. Could you take a look?